create_derivatives/routines.py

```diff
@@ -38,6 +39,8 @@ class BaseRoutine(object):
     def run(self):
         bag = Bag.objects.filter(process_status=self.start_process_status).first()
         if bag:
+            bag.process_status = self.in_process_status
+            bag.save()
             self.process_bag(bag)
```

Comment:
My only concern about this is that if something in `process_bag` blows up the bag will have the `in_process_status` assigned. I think we may need to put that in a try/except block and reset the process status (and re-raise).